tests/unit/config/test_configtypes.py

```diff
@@ -1818,6 +1818,9 @@ def klass(self):
         ({'placeholder': '{}'}, '[foo, "{}", bar]', ['foo', '{}', 'bar']),
         ({'placeholder': '{}'}, '["foo{}bar"]', ['foo{}bar']),
         ({'placeholder': '{}'}, '[foo, "bar {}"]', ['foo', 'bar {}']),
+        ({'placeholder': '{file}'}, '[f, "{file}", b]', ['f', '{file}', 'b']),
+        ({'placeholder': '{file}'}, '["f{file}b"]', ['f{file}b']),
+        ({'placeholder': '{file}'}, '[f, "b {file}"]', ['f', 'b {file}']),
```

Comment:
Hmm, now my own tests got me confused. In `configtypes.py`, `placeholders` is just a boolean, so this should probably actually be `True` instead of `'{}'` or `'{file}'`.